public/scss/mempool.scss

```diff
@@ -0,0 +1,74 @@
+/* mempool */
+.mp-count-card {
+  background-color: #f8f8f9;
+}
+
+.card-pointer {
+  position: absolute;
+  width: 0;
+  height: 0;
+  border-width: 20px;
+  z-index: 10;
+  border-style: solid;
+}
+
+.pointer-right {
+  right: -40px;
+  top: 50%;
+  border-color: transparent transparent transparent white;
+  transform: translateY(-50%);
+}
+
+.pointer-bottom {
+  left: 50%;
+  bottom: -40px;
+  border-color: white transparent transparent transparent;
+  transform: translateX(-50%);
+}
+
+.mempool-tables {
```

Comment:
Instead of creating mempool specific table styles, is there a way we can keep this generic and author in https://github.com/decred/dcrdata/blob/master/public/scss/table.scss ?

Since there are a lot of tables throughout the explorer, we should try to maintain and perfect a core set of styles that we use everywhere instead of subtle differences with associated custom css between the views. 

There's probably even some opportunities to tune the bootstrap variables and get rid of some of the stuff we currently have layered on top of that in table.scss

Reply:
Agree. I'd like to consolidate the table styling in two steps though. I'll move the mempool table CSS to tables.scss and cards.scss here and rename things to be more general, but leave the scoping so that this PR is still just the mempool page. In a follow up PR, I'll hit the table styling and see if we can't get it down to just a few declarations, but would be site-wide.

Reply:
The PR is good for me now. @gozart1?

Reply:
Yep, looking good.